client/my-sites/site-settings/form-base.js

```diff
@@ -89,30 +89,26 @@ module.exports = {
 	 */
 	recordEventOnce( key, eventAction ) {
 		debug( 'record event once: %o - %o', key, eventAction );
-		let newSetting = {};
 		if ( this.state[ 'recordEventOnce-' + key ] ) {
 			return;
 		}
 		this.recordEvent( eventAction );
-		newSetting[ 'recordEventOnce-' + key ] = true;
-		this.setState( newSetting );
+		this.setState( { [ 'recordEventOnce-' + key ]: true } );
 	},
 
 	getInitialState() {
 		return this.getSettingsFromSite();
 	},
 
 	handleRadio( event ) {
-		var name = event.currentTarget.name,
-			value = event.currentTarget.value,
-			updateObj = {};
+		const name = event.currentTarget.name,
+			value = event.currentTarget.value;
 
-		updateObj[ name ] = value;
-		this.setState( updateObj );
+		this.setState( { [ name ]: value } );
```

Comment:
`this.setState( { [ name ]: value } );` doesn't tell me much about the contents of the state. 
maybe switching it to something like 
`currentTargetName` and `currentTargetValue` would be more descriptive.